### DISCUSSION TITLE
Connecting The Ribbon Cable to the TouchScreen Display and FireBeetle2

### DISCUSSION THREAD

**Discussion by daveclifford** [DCC-EX/EX-T3-WiFi#42] [Q&A]
I do not understand how to insert the GDI interface ribbon cable into the Touchscreen Display  GDI interface socket  (DFR0669) and FireBeetle2 (DFR0654-F) GDI socket.  

The interface ribbon has printing on one side and a blue stripe opposite the ribbon connectors on the cable.

1. Do these sockets have a catch that can be opened to insert the cable (then catch is closed) or do you have to use a tweezer to provide pressure to insert the cable?
2. Does the printed side matter on which end to place the cable?

Any clarification or help would be appreciated!!!

Dave Clifford

see [https://darklylabs.com/article/inserting-removing-the-ffc-flat-flexible-cable/] for some info.

see [https://www.ifixit.com/Guide/Recognizing+&+Disconnecting+Cable+Connectors/25629]

**Comment by alex-code** [comment]
I could try and do an assembly video.
The only bit I couldn't really cover is the soldering as I can't undo it to redo it.

The blue bit should be on the brown latch side.
The latch needs to be out then you slot the cable in, holding the cable in you then need to gently push the latch down until it locks.

**Reply by daveclifford** [reply]
I wouldn't worry about soldering  pictures. I think assembly tips (in words) will be sufficient. Showing in sequence how you use the brass ferrule might help. The wiring looks neat and professional. What matters are good soldering joints and connections. Solid wire used to connect the sockets to each other is a must! No stranded wire unless soldered to make the strands into a bundle.

I will try your advice later today.

Dave

Update:  I put on  some magnifiers and voila - I saw the clamp on the Touchscreen! It is amazing what good light and good vision can do!!! Thanks
